images/bazel-tools/build.yaml

```diff
@@ -3,14 +3,18 @@ name: bazel-tools # Name of the image to be built
 variants:
   "10.24":
     arguments:
-      BASE_IMAGE: "eu.gcr.io/jetstack-build-infra-images/golang-dind@sha256:bf41f2a8f6191842ad3ff56a071448ed6a8bdcece4d6d62c5d91733c2f1f3541"
+      BASE_IMAGE: "eu.gcr.io/jetstack-build-infra-images/golang-dind@sha256:a71c5e6e8cb6875e9f6a43b722254693759b5db77566e25ba9eb52ca74b92e08"
+      # Version of Bazel that is bundled in the BASE_IMAGE
+      BAZEL_VERSION: "4.2.1"
+      # Version of Go that is bundled in the BASE_IMAGE
+      GO_VERSION: "1.17"
       NODE_VERSION: "10.24.0~dfsg-1~deb10u1"
       # This DOCKER_TAG is the Docker tag that corresponds to the Node version
       # we use. We don't use the Node version directly because it is not a valid
       # Docker tag.
-      DOCKER_TAG: "10.24.0"
+      NODE_DOCKER_TAG: "10.24.0"
 
 # Image names to be tagged and pushed
 images:
 - ${_REGISTRY}/${_NAME}:${_DATE_STAMP}-${_GIT_REF}-${DOCKER_TAG}
-- ${_REGISTRY}/${_NAME}:latest-${DOCKER_TAG}
+- ${_REGISTRY}/${_NAME}:bazel${BAZEL_VERSION}-go${GO_VERSION}-node${NODE_DOCKER_TAG}
```

Comment:
Maybe just say a bit about how this new tag will be used. 
I can't find any existing references to `latest-${DOCKER_TAG}` in the github.com/jetstack organization.
Will there be a follow up PR which uses this new tag?

Reply:
No, I think this is only used in a [private project](https://github.com/jetstack/fips-builds/blob/master/.github/workflows/builder-image.yaml#L24)... that currently uses sha, but I'd like to change that to use these tags instead so it's easier for me to see what versions of things are currently used.

Reply:
I'll add a note about that there